pkg/pipelines/bootstrap.go

```diff
@@ -40,6 +41,11 @@ type BootstrapOptions struct {
 
 // Bootstrap bootstraps a GitOps pipelines and repository structure.
 func Bootstrap(o *BootstrapOptions, appFs afero.Fs) error {
+	exists, err := ioutils.IsExisting(appFs, filepath.Join(o.OutputPath, pipelinesFile))
+	fmt.Println("This is the path", exists)
+	if exists && !o.Overwrite {
+		return fmt.Errorf("Directory already exists, cannot overwrite ( set --overwrite=true to continue )")
```

Comment:
It is mentioned to check for non-empty output directory.   Sometimes, it is mentioned that pipelines.yaml is checked for existence.   Which one is the intend?    The actual check is on pipelines.yaml.    And the above error can be re-worded to something like:  "XXX already exists.   To overwrite,  please set --overwrite flag and re-run command."

Reply:
We should detect a pipelines.yaml in the directory that the user has provided.

If one exists, then we should not write files, unless --overwrite is set.

Reply:
Error message corrected.< "And the above error can be re-worded to something like: "XXX already exists. To overwrite, please set --overwrite flag and re-run command.">